packages/salesforcedx-vscode-core/src/orgBrowser/metadataType.ts

```diff
@@ -45,15 +46,7 @@ export class TypeUtils {
       telemetryService.sendException('metadata_type_workspace', err);
       throw new Error(err);
     }
-    const workspaceRootPath = workspaceUtils.getRootWorkspacePath();
-    const username = await OrgAuthInfo.getUsername(usernameOrAlias);
-    const metadataTypesPath = path.join(
-      workspaceRootPath,
-      '.sfdx',
-      'orgs',
-      username,
-      'metadata'
-    );
+    const metadataTypesPath = projectPaths.metadataFolder(usernameOrAlias);
```

Comment:
So dumb question.  Can this username or alias ever be different what what we'd get out of the workspaceContextUtil? I did a quick trace of the code flow and it looks like outside of tests this value being passed if read the same way we read it in the WorkspaceContextUtil.  So does it make sense just to refactor here and remove the parameter and always assume the one provided by default?

Reply:
I chatted with @klewis-sfdc and got some context of why this path was followed.  I still think it's the lesser of two evils here to remove the parameter for this method and just use the default value provided in paths vs passing it along.  We'll clean up the value tracked in this class with that follow up ticket to remove the custom listener to changes here in favor of the common one in workspace utils.